sql-indent-left.el

```diff
@@ -0,0 +1,189 @@
+;;; sql-indent-left.el --- configuration options to indent sql -*- lexical-binding: t -*-
+;;
+;; Filename: sql-indent-left.el
+;; Description:
+;; Author: pierre.techoueyres@free.fr
+;; Maintainer: pierre.techoueyres@free.fr
+;; Copyright (C) 2017, Pierre Téchoueyres all rights reserved.
+;; Created:
+;; Version: pierre.techoueyres@free.fr
+;; Last-Updated:
+;;           By:
+;;     Update #: 0
+;; URL:
+;; Keywords: language sql indentation
+;; Compatibility:
+;;
+;; Features that might be required by this library:
+;;
+;;   None
+;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;;; Commentary:
+;;
+;; Set configuration options to indent sql my way.
```

Comment:
You could be more specific here :-)  I would suggest you just have one indentation style defined (the way you use it), with some examples (the example you added to this pull request would do fine).